src/e3/fs.py

```diff
@@ -182,15 +182,19 @@ def find(
     return result
 
 
-def get_filetree_state(path: str, ignore_hidden: bool = True) -> str:
+def get_filetree_state(
+    path: str, ignore_hidden: bool = True, hash_content: bool = False
+) -> str:
     """Compute a hash on a filetree to reflect its current state.
 
     :param path: root path of the file tree to be checked
     :param ignore_hidden: if True (default) then files and directories
         tarting with a dot are ignored.
+    :param hash_content: if True, include the content in the hash.
+
     :return: a hash as a string
 
-    The function will not report changes in the hash if a file is modified
+    The function will not report changes in the hash by default if a file is modified
```

Comment:
To me, the _By default_ should be at the beginning of the phrase : _By default, the function will not report changes in the hash if a file is modified ..._

Reply:
Done